conda-forge.yml

```diff
@@ -7,3 +7,7 @@ provider:
   linux_ppc64le: azure
   win: azure
 test_on_native_only: true
+azure:
+  # toggle for storing the conda build_artifacts directory (including the
+  # built packages) as an Azure pipeline artifact that can be downloaded
+  store_build_artifacts: False
```

Comment:
@maresb, did you want this to be `True`?

Reply:
Oops!!! Ya.

But with the Cleo mess which you just pointed out, it looks like 1.2 is not going to happen.

I would hope that:
* most projects need only `poetry-core` in their host dependencies, and
* the poetry devs stabilize their release schedule in a reasonable timeframe

If those two work out, maybe this isn't a total disaster.